stripped/colabs/Analiza_Korelacji_NASA_stripped.py

```diff
@@ -0,0 +1,122 @@
+"""
+<a href="https://colab.research.google.com/github/PiotrMaciejKowalski/BigData2024Project/blob/CU-86bwcdr5f_Analiza-korelacji-rolinnoci-z-pozostaymi-zmiennymi_Piotr-Puchalski/colabs/Analiza_Korelacji_NASA.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
+"""
+
+"""
+# W tym pliku zajmiemy się analizą korealcji między roślinnością, a innymi cechami zawartymi w naszych danych
+"""
+
+!pip install pyspark
+
+
+
+import pyspark
+from pyspark.sql import SparkSession
+from pyspark.sql.types import IntegerType, FloatType, StringType, StructType
+spark = SparkSession.builder.appName('Corr_Analysis').getOrCreate()
+
+
+"""
+Teraz zaimportujemy dane
+"""
+
+from google.colab import drive
+import pandas as pd
+drive.mount('/content/drive')
+
+sampled = pd.read_csv('/content/drive/MyDrive/BigMess/NASA/sampled_NASA_200k.csv')
+
+schemat = StructType()
+for i in sampled.columns:
+  if i == "Date":
+    schemat = schemat.add(i, StringType(), True)
+  else:
+    schemat = schemat.add(i, FloatType(), True)
```

Comment:
nie podoba mi się podwójne wczytywanie. Załóżmy zadanie na usunięcie z szablonu podwójnego wczytywania pliku.

Reply:
Zrobione

Reply:
Dalej jest podwójne. Patrycja w swoim PR już rozwiązała problem podwójnego wczytywania. (Chodzi o to, że jest pd.read_csv, a potem spark.read)

Reply:
Poprawiłem, żeby było tak jak u Patrycji